README.md

```diff
@@ -1,6 +1,6 @@
 # Java httpbin
 
-[![Maven Central](https://img.shields.io/maven-central/v/org.gaul/httpbin.svg)](https://search.maven.org/#search%7Cga%7C1%7Ca%3A%22httpbin%22)
+[![Build Status](https://travis-ci.org/gaul/java-httpbin.svg?branch=master)](https://travis-ci.org/gaul/java-httpbin) [![Maven Central](https://img.shields.io/maven-central/v/org.gaul/httpbin.svg)](https://search.maven.org/#search%7Cga%7C1%7Ca%3A%22httpbin%22)
```

Comment:
Can you put these on their own lines like https://raw.githubusercontent.com/gaul/s3proxy/master/README.md ?  This will make diffs cleaner and GitHub logically wraps these onto a single line anyway.

I am not opposed to adding this but curious about its value.  None of my projects allow merges of broken pull requests so this should never give any information?  Or is it useful for the periodic build Travis in case some dependency or environment breaks?  Or does it just communicate to users that this project has continuous integration?

Reply:
Change is on own line now (didn't know about github wrapping onto a single line).

For me it's a sign that the repo is healthy and owner is caring enough to have a CI in place.